osgp/platform/osgp-adapter-ws-shared/src/main/java/org/opensmartgridplatform/adapter/ws/domain/entities/NotificationWebServiceLookupKey.java

```diff
@@ -35,6 +41,10 @@ public NotificationWebServiceLookupKey(final String organisationIdentification,
         this.organisationIdentification = Objects.requireNonNull(organisationIdentification,
                 "organisationIdentification must not be null");
         this.applicationName = Objects.requireNonNull(applicationName, "applicationName must not be null");
+        // TODO (RvM): remove lines below
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)